architecture/additional_concepts/storage.adoc

```diff
@@ -168,7 +169,9 @@ spec:
     - name: nfsvol
       mountPath: /scrub
 ----
-<1> However, the particular `server` and `path` values specified in the custom recycler pod template in the `volumes` part is replaced with the particular corresponding values from the PV that is being recycled.
+<1> Namespace where the recycler pod will run. `openshift-infra` is the recommended one, as it already has `pv-recycler-controller` service account that can recycle volumes.
+<2> Name of service account that is allowed to mount NFS volumes. It must exist in the namespace specified by <1>. `pv-recycler-controller` account is recommended, as it is automatically created in `openshift-infra` namespace and has all the required permissions.
```

Comment:
I would see how the reference to <1> in the description for <2> builds. You may need to spell it out.

Reply:
Fixed.